src/test/kotlin/com/pubnub/api/managers/EventEngineManagerTest.kt

```diff
@@ -0,0 +1,11 @@
+package com.pubnub.api.managers
+
+import org.junit.jupiter.api.Test
+
+class EventEngineManagerTest {
+
+    @Test
+    fun `add test here`() {
+        // todo
+    }
+}
```

Comment:
I don't see a point in keeping it as is in the repo

Reply:
Ok. I have just added tests.